Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/ui/details/TracingDetailsItemProvider.kt

```diff
@@ -53,23 +54,27 @@ class TracingDetailsItemProvider @Inject constructor(
                 )
             }.also { add(it) }
 
-            if (latestCalc.riskState != RiskState.CALCULATION_FAILED) {
+            if (latestCalc.riskState != RiskState.CALCULATION_FAILED && status != Status.TRACING_INACTIVE) {
                 PeriodLoggedBox.Item(
                     activeTracingDaysInRetentionPeriod = activeTracingDaysInRetentionPeriod.toInt()
                 ).also { add(it) }
             }
 
-            when (latestCalc.riskState) {
-                RiskState.LOW_RISK -> DetailsLowRiskBox.Item(
-                    riskState = latestCalc.riskState,
-                    matchedKeyCount = latestCalc.matchedKeyCount
-                )
-                RiskState.INCREASED_RISK -> DetailsIncreasedRiskBox.Item(
-                    riskState = latestCalc.riskState,
-                    lastEncounteredAt = latestCalc.lastRiskEncounterAt ?: Instant.EPOCH
-                )
-                RiskState.CALCULATION_FAILED -> DetailsFailedCalculationBox.Item
-            }.also { add(it) }
+            if (status == Status.TRACING_INACTIVE) {
```

Comment:
What about

```kotlin
    when {
        status == Status.TRACING_INACTIVE || latestCalc.riskState == RiskState.CALCULATION_FAILED -> {
            DetailsFailedCalculationBox.Item
        }
        latestCalc.riskState == RiskState.LOW_RISK -> DetailsLowRiskBox.Item(
            riskState = latestCalc.riskState,
            matchedKeyCount = latestCalc.matchedKeyCount
        )
        latestCalc.riskState == RiskState.INCREASED_RISK -> DetailsIncreasedRiskBox.Item(
            riskState = latestCalc.riskState,
            lastEncounteredAt = latestCalc.lastRiskEncounterAt ?: Instant.EPOCH
        )
        else -> null
    }?.let { add(it) }
```

Reply:
Was thinking of this variant at first (as it looks less nested), thought that it would be explicitly defined for inactive tracing. But nvm - already switched to it.